tests/no-internal.js

```diff
@@ -34,8 +35,10 @@ ruleTester.run(
   NoInternalESLintRule,
   supportSkippedAndOnlyInTests({
     valid: [
-      { code: `import { internal, public, Internal, Public } from "test-pkg-1";` }, // not a bentley/itwin scope
```

Comment:
Do we not need this valid case anymore?

Reply:
No, it is identical to [this case](https://github.com/iTwin/eslint-plugin/pull/32/files/a101786c814ae8102bc84f7964d938fdc2765598#diff-ad2d99451832c5234ef610cea43b09e226f2c04990f1e499018be8dd621965c4R43-R53) except without actually calling the imported functions. I noticed that if they aren't actually called, no error is thrown by the linter so I think this first case in unnecessary.